README.md

```diff
@@ -4,11 +4,29 @@ Creating an Elo system for our local Darts matches/tournaments.
 The Aim with this project is to creating a functional Elo system similar to one which can be found in games such as chess and Go for Darts.
 The product must be easy to use and accessible for non technical users for use in our local darts matches which take place during our tournaments. 
 
-Special thanks to Daniel Ryan for providing the R code and mathematical expertisehis original product can be seen in the ExcelSheet.R file and Coolmoyne Leaderboards.xlsx excel file. 
+## Download instructions
+Run `git clone https://github.com/jackconryan/DartsElo`
 
+Run `npm install` to download dependencies
```

Comment:
npm is not relevant to this project it is used for Javascript and node.js which is not present in the code in its current state.